lib/nexpose/site.rb

```diff
@@ -617,6 +617,14 @@ class IPRange
     def initialize(from, to = nil)
```

Comment:
And as to documentation, it should include a mention that this is different from how Nexpose interprets CIDR notation when entered in the UI. In the UI, a CIDR notation gets stripped of its first and last IP in the range. (The UI used to have a message about it, but I don't see it now after the site refactor. But when I test it, it still behaves that way.)

The other alternative is to make it behave like the UI does. Users could still work around it by providing an explicit range.